.github/actions/bumpVersion/functions.js

```diff
@@ -0,0 +1,75 @@
+const utils = require('util');
```

Comment:
You should move this file to `.github/libs/versionUpdater.js`, and consider merging it with `.github/libs/nativeVersionUpdater.js`